public/styles.css

```diff
@@ -1073,6 +1073,7 @@ input[type=number]::-webkit-outer-spin-button {
 .mobile-content .trade-params > div {
     margin-right: .5rem;
     min-width: 10rem;
```

Comment:
I think we shouldn't set the min-width here , It stretch the control where not required and tend to compensate with spaces where the control cannot fit the req. I mentioned it my new PR . I am of the opinion that in this case , we set for a control individually where necessary.